### PR TITLE
Gale mandarin fix

### PR DESCRIPTION
**Pull request by naxingyu**
This PR is based on Yenda's #1207 for addressing #1163

### CODE REVIEW

**Comment by danpovey** [comment]
Is this ready to review, or you are still working on it?

**Comment by naxingyu** [comment]
Still working on it.

**Comment by naxingyu** [comment]
@danpovey Ready.

File: egs/gale_mandarin/s5/run.sh [hunk 1]
```diff
@@ -6,62 +6,87 @@
 . ./path.sh
 . ./cmd.sh
 
-nJobs=40
-nDecodeJobs=40
-
-AUDIO_PATH=/export/corpora5/LDC/LDC2013S08/
-TEXT_PATH=/export/corpora5/LDC/LDC2013T20/
-
+nJobs=64
```
Comment (danpovey) [inline egs/gale_mandarin/s5/run.sh]: could you please rename these variables to num_jobs and num_jobs_decode ?

Comment (naxingyu) [inline egs/gale_mandarin/s5/run.sh]: Done.

**Comment by naxingyu** [comment]
This PR is for addressing #1163 . I'll created the pitch example using hkust, which is not so heavy to train.

**Comment by danpovey** [comment]
OK, thanks!   Merging.

**Pull request merged**